sql-cli/sql_cli/configuration.py

```diff
@@ -94,3 +92,16 @@ def write_value_to_yaml(self, section: str, key: str, value: str, filepath: Path
 
         with filepath.open(mode="w") as fp:
             yaml.dump(yaml_config, fp)
+
+    def to_dict(self) -> dict[str, Any]:
+        """
+        Return configuration represented as JSON string.
+
+        :returns: Dictionary containing non-sensitive global and per-environment configuration.
+        """
+        config_as_dict = {"global": self.from_yaml_to_dict(self.get_global_config_filepath())}
+        if self.environment:
+            env_config = self.from_yaml_to_dict(self.get_env_config_filepath())
+            env_config.pop("connections", None)  # remove sensitive information
+            config_as_dict[self.environment] = env_config
```

Comment:
```suggestion
            config_as_dict["local"] = env_config
```
Would it make sense to rename the key to "local", or the requestor will know how exactly to parse the response for environment values?

Reply:
@pankajkoti, at the moment, it returns for all environments - and I'd expect the one using the endpoint to know which environment they are interested in. Do you see challenges with this approach?